chainer/training/extensions/log_report.py

```diff
@@ -0,0 +1,115 @@
+import json
+import os
+import tempfile
+
+import six
+
+from chainer import reporter
+import chainer.serializer as serializer_module
+from chainer.training import extension
+import chainer.training.trigger as trigger_module
+
+
+class LogReport(extension.Extension):
+
+    """Trainer extension to output the accumulated results to a log file.
+
+    This extension accumulates the observation of the trainer to
+    :class:`~chainer.DictSummary` for each duration of a specified length, and
```

Comment:
Consider re-wording 'for each duration of a specified length' for clarity. Maybe 'at a regular interval specified by a supplied trigger'?